src/Link/index.js

```diff
@@ -1,23 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { style } from 'styled-system';
+import { system } from 'styled-system';
+import Box, { StyledBox } from '../Box';
 import theme from '../theme';
-import Text from '../Text';
 
-const hoverColor = style({
-  prop: 'hoverColor',
-  cssProperty: 'color',
-  key: 'colors',
+const hoverColor = system({
+  hoverColor: {
+    property: 'color',
+    scale: 'colors',
+  },
 });
 
-const activeColor = style({
-  prop: 'activeColor',
-  cssProperty: 'color',
-  key: 'colors',
+const activeColor = system({
+  activeColor: {
+    property: 'color',
+    scale: 'colors',
+  },
 });
 
-const StyledLink = styled(Text)`
+const StyledLink = styled(StyledBox)`
```

Comment:
Not a deal breaker at all - but wondering about your thoughts on why this change was made. Do we not want a Link to act like Text? I can wrap `<Text><Link>` and do things like `caps` and `italics` to get the old text effects applied to a Link so I don't think we've lost any functionality.

Reply:
I don't really want people to use those props anymore actually. Just left them there to not break things for now.

You can use `fontStyle={'italic'}` if you want to change the font style. Setting `italics` is just the old way of doing it. 

Having `caps` map to  `text-transfrom: uppercase` doesn't make much sense to me anymore. Why wouldn't it map to `text-transfrom: capitalize` for that matter? I think we should remove it. I don't really know of anyone using it either way. If anything, encouraging the use of that text transform can introduce accessibility issues more that it can solve anything. 

https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

Reply:
No problem